src/components/EventCard/EventCard.js

```diff
@@ -143,6 +157,7 @@ EventCard.propTypes = {
 	location: PropTypes.string,
 	// link might not be available yet
 	detailLink: PropTypes.string,
+	zoomLink: PropTypes.string,
```

Comment:
I'd rather this be known as "conferenceLink", just to be vendor-neutral.

Reply:
Waaht, but we are zoomers.

But also yes I agree I'll fix this 👍